### PR TITLE
Implemented getters and setters on 3d layout layer properties

### PR DESCRIPTION
**Pull request by maxcapodi78**
#1592

### CODE REVIEW

**Review by Samuelopez-ansys — APPROVED** [review]
LGTM

File: pyaedt/modules/LayerStackup.py [hunk 1]
```diff
@@ -143,10 +143,616 @@ def __init__(self, app, layertype="signal", negative=False):
...
-        self.usp = False
+        self._usp = False
         self.hfssSp = {"si": True, "dt": 0, "dtv": 0.1}
         self.planaremSp = {"ifg": False, "vly": False}
-        self.zones = []
+        self._zones = []
+
+    @property
+    def color(self):
+        """Get/Set the property to the active layer. Color can be an integer or a list of rgb values (0,255).
```
Comment (MaxJPRey) [inline pyaedt/modules/LayerStackup.py:153]:
```suggestion
        """Get/Set the property of the active layer. Color can be an integer or a list of rgb values (0,255).
```

File: pyaedt/modules/LayerStackup.py [hunk 2]
```diff
@@ -143,10 +143,616 @@ def __init__(self, app, layertype="signal", negative=False):
...
+        Returns
+        -------
+        str
+        """
+        return self._toprounghenss
+
+    @toprounghenss.setter
+    def toprounghenss(self, val):
+        self._toprounghenss = val
+        self.update_stackup_layer()
```
Comment (MaxJPRey) [inline pyaedt/modules/LayerStackup.py:438]:
```suggestion
    def top_roughness(self):
        """Get/Set the active layer top roughness (with units).

        Returns
        -------
        str
        """
        return self._top_roughness

    @top_roughness.setter
    def top_roughness(self, val):
        self._top_roughness= val
        self.update_stackup_layer()
```

File: pyaedt/modules/LayerStackup.py [hunk 3]
```diff
@@ -114,25 +114,25 @@ def __init__(self, app, layertype="signal", negative=False):
...
-        self.index = 1
+        self._locked = False
+        self._topbottom = "neither"
+        self._pattern = 1
+        self._drawoverride = 0
+        self._thickness = 0
+        self._lowerelevation = 0
+        self._roughness = 0
+        self._botroughness = 0
+        self._toprounghenss = 0
```
Comment (MaxJPRey) [inline pyaedt/modules/LayerStackup.py:125]:
```suggestion
        self._top_roughness= 0
```

File: pyaedt/modules/LayerStackup.py [hunk 4]
```diff
@@ -143,10 +143,616 @@ def __init__(self, app, layertype="signal", negative=False):
...
+    def toprounghenss(self):
+        """Get/Set the active layer top roughness (with units).
+
+        Returns
+        -------
+        str
+        """
+        return self._toprounghenss
+
+    @toprounghenss.setter
```
Comment (MaxJPRey) [inline pyaedt/modules/LayerStackup.py:435]:
```suggestion
    @top_roughness.setter
```

Comment (maxcapodi78) [reply pyaedt/modules/LayerStackup.py:452]: @MaxJPRey the reason I left those names is to keep the back compatibility. their name was this before and If I change I risk to break it. Any suggestion?

File: pyaedt/modeler/Modeler.py [hunk 5]
```diff
@@ -1374,7 +1374,7 @@ def _get_lists_data(self):
                             props["List"] = data["GeometryEntityListParameters"]["EntityList"]
                         design_lists.append(Lists(self, props, name))
             except:
-                self.logger.error("Lists were not retrieved from AEDT file")
+                self.logger.infi("Lists were not retrieved from AEDT file")
```
Comment (MaxJPRey) [inline pyaedt/modeler/Modeler.py:1377]:
```suggestion
                self.logger.info("Lists were not retrieved from AEDT file")
```

File: pyaedt/modules/LayerStackup.py (hunk 4 above)
Comment (MaxJPRey) [reply pyaedt/modules/LayerStackup.py:452]: Could we have both properties (`top_roughness` and `toprounghenss`)
The old one could return the value of the new one.
For instance for the getter of the old one,
```
@toprounghenss
def toprounghenss (self):
    return self.top_roughness
```

For the setter of the old one:
```
  @top_roughness.setter
  def top_roughness(self, val):
      self._top_roughness= val
      self.update_stackup_layer()
```

And so we don't expose the protected attribute `_toprounghenss` anymore.

Comment (maxcapodi78) [reply pyaedt/modules/LayerStackup.py]: I've made all exposed properties pythonic

Comment (MaxJPRey) [reply pyaedt/modules/LayerStackup.py]: @maxcapodi78 Perfect! Thanks

**Review by MaxJPRey — APPROVED** [review]

**Pull request merged**